docs/content/configuration/indexing-service.md

```diff
@@ -25,6 +25,15 @@ If you are running the indexing service in remote mode, the task logs must be st
 |--------|-----------|-------|
 |`druid.indexer.logs.type`|Choices:noop, s3, azure, hdfs, file. Where to store task logs|file|
 
+You can also configure the Overlord to automatically retain the task logs only for last x seconds by configuring following additional properties.
+
+|Property|Description|Default|
+|--------|-----------|-------|
+|`druid.indexer.logs.kill.enabled`|Boolean value for whether to enable deletion of old task logs. |false|
+|`druid.indexer.logs.kill.durationToRetain`| Required if kill is enabled. In seconds, task logs to be retained created in last x seconds. |None|
```

Comment:
default does not matter because it is forced to be provided by the user when enabled. Long.MAX_VALUE is there just because some value needs to be stored there when it is not enabled.